### DISCUSSION TITLE
Input prohibition

### DISCUSSION THREAD

**Discussion by foxter111** [JedWatson/react-select#4738] [Q&A] [answered]
Hello. Is it possible to prohibit the entry of numbers into the input?

**Comment by Rall3n** [comment, marked as answer]
Hello @foxter111,

I hope you don't mind me converting your issue into a discussion, as this is a more appropriate place for questions.

In its current state you can't directly prohibit the input of numbers into the `Select` component (by using `<input pattern="">`).
But you can remove numbers from the input by using the `onInputChange` prop.

```jsx
<Select
  onInputChange={(inputValue) => {
    let value = inputValue.replace(/\d/g, '');

    // Either setState when controlled
    setInputValueState(value);
    // or return for uncontrolled.
    return value;
  }}
/>
```

[CodeSandbox example](https://codesandbox.io/s/react-select-v4-no-numbers-kowwq?file=/example.js)

**Reply by foxter111** [reply]
@Rall3n thank you. it`s working.

**Reply by Rall3n** [reply]
@foxter111 Please don't forget to mark as an answer
